Doctrine/Orm/DataProvider.php

```diff
@@ -30,13 +30,13 @@ class DataProvider implements DataProviderInterface
     private $managerRegistry;
 
     /**
-     * @var QueryExtensionInterface[]
+     * @var QueryCollectionExtensionInterface[]|QueryItemExtensionInterface[]
```

Comment:
This is a wrong docblock typehint, but help IDE.

However, as we cannot typehint on method now, do we need another (empty) interface ?...